docs/DeveloperGuide.md

```diff
@@ -312,16 +312,79 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `TutorTrack`, the **Actor** is the `Tutor` and the **Person** is the `Tutee`, unless specified otherwise)
+
+**Use case: View all Person**
+
+**MSS**
+
+1.  User requests to list all persons
+2.  System shows a list of all persons
+
+    Use case ends.
+
+**Extensions**
+* 2a. The list is empty.
+* *a. Connection lost.
+    * *a1. System informs the user.
+
+  Use case ends.
+
+**Use case: Add a Person**
+
+**MSS**
+
+1.  User requests to list all persons
+2.  System shows a list of all persons
+3.  User requests to add a specific person in the list
+4.  System adds the person
+
+    Use case ends.
+
+**Extensions**
+* 3a. The given format is invalid.
+    * 3a1. System shows an error message.
+
+      Use case resumes at step 2.
+* *a. Connection lost.
+    * *a1. System informs the user.
+
+      Use case ends.
+
+**Use case: Update a Person**
+
+**MSS**
+
+1.  User requests to list all persons
+2.  System shows a list of all persons
+3.  User requests to update a specific person in the list
+4.  System update the person
+
+    Use case ends.
+
+**Extensions**
+* 2a. The list is empty.
+* 3a. The given format is invalid.
```

Comment:
I think instead of use case resuming at step 2., it could be something like
```suggestion
* 3a. The given format is invalid.
    * 3a1. TutorTrack shows an error message.
    * 3a2. TutorTrack requests for the correct data
    * 3a3. Tutor enters new data
  
    Steps 3a2-3a3 are repeated until the data entered are correct.
  
    Use case resumes from step 4.
```

because otherwise it seems to be suggesting that the app will display a list again if the user enters a wrong format.